pcdsdevices/lens.py

```diff
@@ -20,7 +18,10 @@
                         pseudo_position_argument, real_position_argument)
 from .sim import FastMotor
 
-LENS_RADII = [50e-6, 100e-6, 200e-6, 300e-6, 500e-6, 1000e-6, 1500e-6]
+logger = logging.getLogger(__name__)
+
+# Set of Be lenses with thicknesses.
+LENS_RADII = calcs.LENS_RADII
```

Comment:
Is this used in the file now that the calcs are all in pcdscalcs?

Reply:
Ohh good point, we're not using this here anymore, ('ill double check) but we should not need to use it at least - thank you

Reply:
took it out in d3c2985